examples/3dsphere/baroclinic_wave.jl

```diff
@@ -294,6 +294,16 @@ function rhs!(dY, Y, _, t)
     @. dρe -= vdivf2c(fw * Ic2f(cρe))
     @. dρe -= vdivf2c(Ic2f(cuₕ * cρe))
 
+    # 4) flux correction
+    fcc = Operators.FluxCorrectionC2C(
+        bottom = Operators.Extrapolate(),
+        top = Operators.Extrapolate(),
+    )
+
+    @. dρ += fcc(fw, cρ)
+    @. dρe += fcc(fw, cρe)
+    # @. duₕ += fcc(cw, cρ*uₕ)/cρ
```

Comment:
@simonbyrne Is this the way to implement density weighted correct for horizontal velocity?

Reply:
```suggestion
    # @. duₕ += fcc(cw, cρ*cuₕ)/cρ
```